packages/@lwc/engine-server/src/__tests__/fixtures/scoped-slots/mixed-with-light-dom-slots-outside/expected.html

```diff
@@ -0,0 +1,17 @@
+<x-parent>
+  <template shadowrootmode="open">
+    <x-child>
+      <div>
+        <!---->
+        <!---->
+        <!---->
+        <!---->
+        <!---->
+        <!---->
+        <!---->
+        Fallback slot content 3
+        <!---->
+      </div>
+    </x-child>
+  </template>
+</x-parent>
```

Comment:
This one is bizarre and probably a bug.

Reply:
Actually wait, it makes sense, the child has no non-scoped slots, so all the default-slotted content is just ignored.